packages/salesforce/src/Adaptor.js

```diff
@@ -306,8 +307,8 @@ const defaultOptions = {
  * @function
  * @param {String} qs - A query string.
  * @param {Object} options - Options passed to the bulk api.
- * @param {integer} [options.pollTimeout=90000] - Polling timeout in milliseconds.
- * @param {integer} [options.pollInterval=3000] - Polling interval in milliseconds.
+ * @param {Integer} [options.pollTimeout=90000] - Polling timeout in milliseconds.
```

Comment:
Why the case change here? I would expect a primitive `integer` to be lower case.  In the case of `Boolean`, `Boolean` is a class name in Javascript - so do we mean `new Boolean(true)` or `true`?

I would set them all to lowercase unless there's a compelling reason for the change

Reply:
I wanted consistency, i think it should be consistency for all adaptor. I will change to lower case

Reply:
I have rename all params type to be lower case ✅ , with exception of `RequestOptions`

Reply:
Good. Any class-like type should be ProperCased, but a primitive should be lower case. I think that's what we usually do, but I'm sure there are inconsistencies. Not a big deal!

Reply:
Ah, wait, Operation should be capital O

Reply:
@josephjclark so for `Object` and `Array` they should be ProperCased right ?

Reply:
All I said is that `Operation` should be have a captital O, because there's a specific type interface to an Operation. It has a formal definition.

For the record, object should probably be lowercase because it's an informal definition (we provide no type interface for the shape of the object, it's just an object).

Array can go both ways because you can use the term informally. For consistency's sake we should probably capitalise it.

I'm starting to  formalise some of this stuff into a style guide. Maybe it'll help encourage consistency (eventually)